src/Congo.Contracts/Responses/Cart/CartResponse.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+
+namespace Congo.Contracts.Responses.Cart
+{
+    public sealed class CartResponse
+    {
+        public Guid Id { get; init; }
+
+        public ICollection<CartItemResponse> CartItems { get; init; }
```

Comment:
Can be of type IEnumerable since we shouldn't be modifying this list

Reply:
For this particular case, IEnumerable seems like the correct choice. But if you really want to "show" your intention for a readonly list, there is `IReadOnlyCollection<T>`.